src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets

```diff
@@ -198,7 +199,7 @@
 	as nuget packages content.
 	-->
 	<Target Name="_UnoAssetsGetCopyToPublishDirectory"
-	BeforeTargets="GetCopyToPublishDirectoryItems">
+			BeforeTargets="GetCopyToPublishDirectoryItems;_UnoGeneratePriMarker">
```

Comment:
I have already tried it but it didn't work at that time for some reason. https://github.com/unoplatform/uno/pull/13156 but it broke template tests on CI.

Reply:
Thanks, will validate further!